example-app/src/main/java/com/adyen/checkout/example/ui/card/SessionsCardUiState.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 30/11/2023.
+ */
+
+package com.adyen.checkout.example.ui.card
+
+import androidx.compose.runtime.Immutable
+import com.adyen.checkout.card.CardConfiguration
+import com.adyen.checkout.components.core.action.Action
+import com.adyen.checkout.example.ui.compose.ResultState
+
+@Immutable
+internal data class SessionsCardUiState(
+    val cardConfiguration: CardConfiguration,
+    val isLoading: Boolean = false,
+    val toastMessage: String? = null,
```

Comment:
`toastMessage` is maybe not the right name, because `toast` is an implementation detail. I want to show a one time message here, so it's not really a state. It's actually an event. Also it's not only for errors, but for any sort of message event.

Maybe just renaming the field and using a type alias for String would make this better?